core/DataAccess/RawLogDao.php

```diff
@@ -225,7 +225,7 @@ public function hasSiteVisitsBetweenTimeframe($fromDateTime, $toDateTime, $idSit
         $sites = Db::fetchOne("SELECT 1
                 FROM " . Common::prefixTable('log_visit') . "
                 WHERE idsite = ?
-                AND visit_last_action_time > ?
+                AND visit_last_action_time >= ?
```

Comment:
Explanation part 10 (side note): found an issue here in the code I think. Matomo will look for data within eg, 2012-02-02 00:00:00 => 2012-02-03:00:00, but using `>`, means we won't get anything if the visit happens on `'2012-02-02 00:00:00'`. Edge case, but it should be handled.